example/methods_directory/physical/psp.json

```diff
@@ -1,55 +1,98 @@
 {
     "name": "PAW Pseudopotential method",
     "path": "/qm/wf/psp/paw",
-    "slug": "pseudopotential",
     "categories": {
         "tier1": "qm",
         "tier2": "wf",
         "type": "psp",
         "subtype": "paw"
     },
-    "data": [
+    "metaProperties": [
```

Comment:
It's good to rename this, but we don't necessarily need to - it will require changes to JS/webapp. Are those changes present as well?

Reply:
I think currently we do not have equivalent of `psp` schema, we are only using `psp/file` in `imports/properties/schemas/pseudo.js`.